src/commands/synthetics/utils.ts

```diff
@@ -455,16 +455,62 @@ export const getReporter = (reporters: Reporter[]): MainReporter => ({
   },
 })
 
+const getTest = async (api: APIHelper, {id, suite}: TriggerConfig): Promise<{test: Test} | {errorMessage: string}> => {
+  try {
+    const test = {
+      ...(await api.getTest(id)),
+      suite,
+    }
+
+    return {test}
+  } catch (error) {
+    if (isNotFoundError(error)) {
+      const errorMessage = formatBackendErrors(error)
+
+      return {errorMessage: `[${chalk.bold.dim(id)}] ${chalk.yellow.bold('Test not found')}: ${errorMessage}`}
+    }
+
+    throw error
+  }
+}
+
+const getTestAndOverrideConfig = async (
+  api: APIHelper,
+  {config, id, suite}: TriggerConfig,
+  reporter: MainReporter,
+  summary: Summary
+) => {
+  id = PUBLIC_ID_REGEX.test(id) ? id : id.substr(id.lastIndexOf('/') + 1)
```

Comment:
I'd be in favor of creating a new variable, instead of mutating the one extracted from the second parameter 🤔  Something such as:

```ts
// or `parsedId`, or `sanitizedId`, or `formattedId`...
const normalizedId = PUBLIC_ID_REGEX.test(id) ? id : id.substr(id.lastIndexOf('/') + 1)
```